daml-script/daml3/Daml/Script/Questions/Submit/Error.daml

```diff
@@ -114,6 +117,15 @@ data SubmitError
     ValueNesting with
         limit : Int
           -- ^ Nesting limit that was exceeded
+  | -- | The transaction refers to locked contracts which are in the process of being created, transferred, or
+    -- archived by another transaction. If the other transaction fails, this transaction could be successfully retried.
+    LocalVerdictLockedContracts with
+      localVerdictLockedContracts : [AnyContractId]
+          -- ^ Locked contract ids
+  | -- | The transaction refers to locked keys which are in the process of being modified by another transaction.
+    LocalVerdictLockedKeys with
+      localVerdictLockedKeys : [AnyContractKey]
+          -- ^ Locked contract keys
```

Comment:
that's a fair point. In the case of `LOCAL_VERDICT_LOCKED_KEYS` I know that the template id is available because the error message contains a stringified `GlobalKey` (see https://github.com/digital-asset/daml/pull/17608#discussion_r1368837577). I'm not sure about `LOCAL_VERDICT_LOCKED_CONTRACTS` because the error message drops the contract ids (see https://github.com/digital-asset/daml/pull/17608#discussion_r1368837516), but I think that's because of a bug in Canton* and not because the template id is truly missing

\* In [(canton repo) `LocalVerdict.scala`](https://github.com/DACH-NY/canton/blob/main/community/base/src/main/scala/com/digitalasset/canton/protocol/messages/LocalVerdict.scala#L351-L391), `LockedKeys.Reject` and `LockedContracts.Reject` both extend `LocalRejectImpl`, but the former sets `_details` while the latter doesn't.